main.go

```diff
@@ -123,6 +144,19 @@ func main() {
 		LeaderElectionID:       "appmesh-controller-leader-election",
 		HealthProbeBindAddress: healthProbeBindAddress,
 	})
+
+	converter := conversions.NewPodConverter()
+
+	customController := k8s.NewCustomController(
+		clientSet,
+		listPageLimit,
+		metav1.NamespaceAll,
+		converter,
```

Comment:
nit: using conversions.NewPodConverter() directly would be cleaner I think. But confirm with Yang once.